tchannel/container/heap.py

```diff
@@ -122,3 +136,42 @@ def down(h, parent, n):
 
         h.swap(parent, min_child)
         parent = min_child
+
+
+def smallest(h, p, current=0):
```

Comment:
ooh i forgot that `current=0` isn't actually used.

regarding naming: I was trying to be consistent with the rest of the file but I'll expand them.